lib/whitehall_importer/integrity_checker.rb

```diff
@@ -2,6 +2,15 @@ module WhitehallImporter
   class IntegrityChecker
     attr_reader :edition
 
+    def self.time_similar?(proposed_time, publishing_api_time)
```

Comment:
I think we should probably call this matches rather than similar. I'm assuming you went for similar as we're not checking it's exact but I think match still conveys are meaning better than similar. 

A similar time to me sounds like there's a pretty big window for difference whereas we're right down to the 10 second mark here.

Reply:
Sure, I can see where you're coming from but I suppose it's a matter of opinion, you could also say that 'matches' means two things that are equal which isn't really true if the times differ slightly. To keep the PR moving I've adjusted the name as per your suggestion.